packages/wallet/src/SingleAddressWallet.ts

```diff
@@ -125,6 +125,7 @@ export class SingleAddressWallet implements ObservableWallet {
   readonly util: WalletUtil;
   readonly rewardsProvider: TrackedRewardsProvider;
 
+  // eslint-disable-next-line max-statements
```

Comment:
Please help with an alternate solution to too many statements in constructor

Reply:
I think it's reasonable to merge with this, as reworking the interface is out of scope, and there will be some attention here when refactoring to support the narrowed types.

Reply:
This is reasonable to leave since refactoring the constructor is better left for the planned rework relating to narrowed internal types.